boxsdk/client/client.py

```diff
@@ -6,18 +6,20 @@
 from ..config import API
 from ..session.box_session import BoxSession
 from ..network.default_network import DefaultNetwork
-from ..object.user import User
+from ..object.cloneable import Cloneable
 from ..object.folder import Folder
 from ..object.search import Search
 from ..object.events import Events
 from ..object.file import File
 from ..object.group import Group
 from ..object.group_membership import GroupMembership
+from ..object.user import User
```

Comment:
You should merge master. Not only is your PR not mergeable right now, but I also think that this import is no longer used on master.